### PR TITLE
Adds Valkey 8.0.0-rc2 files and worflow changes to ignore some file changes.

### PR DESCRIPTION
**Pull request by roshkhatri**
Adds Valkey 8.0.0-rc2 files and workflow changes to ignore some file changes and  adds a manual option to update the description so we do not need to build the containers for smaller changes unless changes in the Dockerfiles.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -3,6 +3,9 @@ name: GitHub CI
 on:
   pull_request:
   push:
+    paths-ignore:
+      - '**.md'
+      - '**.sh'
```
Comment (hpatro) [inline .github/workflows/ci.yml]: Why do we need double asterisk ?

Comment (roshkhatri) [reply .github/workflows/ci.yml]: Its the yml syntax for any files with the extensions in github workflow files

I have tested it that it ignores in my personal repo: https://github.com/roshkhatri/valkey-container/commit/946e48a6cdae773f10898a0294ee64ae84097a25
where it ignores the changes

**Review by hpatro — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (hpatro) [reply .github/workflows/ci.yml]: Does `*.md` works ?

**Comment by roshkhatri** [comment]
I think I should add the path to run only for changes in `Dockerfiles`

**Review by madolson — APPROVED** [review]

**Pull request merged**